flask_smorest/utils.py

```diff
@@ -137,3 +137,38 @@ def prepare_response(response, spec, content_type):
                         field
                     ]
                 ) = response.pop(field)
+
+
+def normalize_config_prefix(config_prefix: str):
+    """Normalize custom API config prefix
+
+    :param str config_prefix: raw prefix
+    :return: normalized prefix. Underscore is automatically added if prefix misses one.
+    """
+    result = config_prefix.strip().upper()
+    if result and not result.endswith("_"):
+        result += "_"
+    return result
+
+
+def get_config_key(*, ctx, key):
```

Comment:
Why kw only args?

Reply:
Main reasoning is for `get_config_value`. There I am forcing kw so it has similar interface as `get_config_value`.